### DISCUSSION TITLE
AutoBudget Via CronJob Doesn't Work . FireflyIII : 6.1.9

### DISCUSSION THREAD

**Discussion by parthapratimpalit** [firefly-iii/firefly-iii#8533] [Firefly III]
Hello Team,

**My Installation As below**

Proxmox Server
Container Image : Ubantu  22.04
PHP : 8.3
Apache2
Firefly Version : 6.1.9

I tried multiple permutations and combination and gone through all the chats but still not able to make the Cronjob to auto fill up the Budget for next months. Once i fill the Budget for one month the next month it vanishes and becomes 0. 

**Sample Below**
=================
**Feb 2024 Budget**

![image](https://github.com/firefly-iii/firefly-iii/assets/99000666/1dd7764c-3ed2-4b6a-ace3-785480c31b94)
![image](https://github.com/firefly-iii/firefly-iii/assets/99000666/70c9e896-ccea-400e-8209-8b005dff02e9)

**FireFlyIII Installation Folder**

![image](https://github.com/firefly-iii/firefly-iii/assets/99000666/b0a7035e-8d19-4d78-b243-b41f58588a25)

**PHP Location**

![image](https://github.com/firefly-iii/firefly-iii/assets/99000666/4cb97f47-2215-42d8-af76-f8f3566e5905)

Tried Below Cron Job

**Attempt 1**
![image](https://github.com/firefly-iii/firefly-iii/assets/99000666/07c94261-fc61-49c8-a7ed-5ba343f398bc)

**Attempt 2**
![image](https://github.com/firefly-iii/firefly-iii/assets/99000666/7216d0ec-ead7-4992-8e93-412b62117404)

**Attempt 3**
![image](https://github.com/firefly-iii/firefly-iii/assets/99000666/c5b9bedb-6daa-4d1f-ade7-e5178d5d700d)

**Attempt 4**
![image](https://github.com/firefly-iii/firefly-iii/assets/99000666/c50a7ef6-9d98-4c6e-b14c-c033f45364bc)

**Attempt 5**

Manual Hit
![image](https://github.com/firefly-iii/firefly-iii/assets/99000666/3fea7e5a-f761-4559-81da-5a02171d3a6e)

But still the March and later month auto budget doesn't get filled up

![image](https://github.com/firefly-iii/firefly-iii/assets/99000666/520ea46c-cbf1-42c0-ad84-375d6ba5f9df)

**Comment by JC5** [comment]
https://docs.firefly-iii.org/how-to/firefly-iii/finances/budgets/#automated-budgets

Check out the table. Auto-budgets are only created on the first of the month.
